packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,322 @@
+- **afterLoad** `(shippingMethodsResponse => shippingMethodsResponse.shippingMethods)` - called after loading shipping methods
+- **beforeSelect** `(shippingMethod => shippingMethod)` - called before selecting shipping method
+- **afterSelect** `(selectedShippingMethod => void)` - called after selecting shipping method
+- **beforeSelectedDetailsChange** `(details => details)` - called before modifying currently picked shipping method, e.g. selecting parcel locker on the map
+- **afterSelectedDetailsChange** `(details => void)` - called after modifying currently picked shipping method
+- **onError** `(({ action, error }) => void)` - called when some operation throws an error
+
+```vue
+<VsfShippingProvider
+  :beforeLoad="beforeLoad"
+  :afterLoad="afterLoad"
+  :beforeSelect="beforeSelect"
+  :afterSelect="afterSelect"
+  :beforeSelectedDetailsChange="beforeSelectedDetailsChange"
+  :afterSelectedDetailsChange="afterSelectedDetailsChange"
+  :onError="onError"
+/>
+```
+## Collecting and saving billing details
+Billing details are information about the payer and her/his address. Based on that, payment providers can evaluate probability of fraud payment and store owners can prepare invoices.
+
+To access billing details, we can use property of `useBilling` called `billing`.
+```js{8,16}
+import { useBilling } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      billing
+    } = useBilling();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      billing
+    };
+  }
+}
+```
+`billing` property will return `null` if the address is not saved on the server or has not been loaded.   
+
+In order to save billing details on the server, you could use `save` method:
+```vue{2,15,24}
+<template>
+  <form @submit.prevent="save({ billingDetails: billingForm })">
+    <!-- form fields -->
+    <button type="submit" :disabled="loading">Submit</button>
+  </form>
+</template>
+
+<script>
+import { useBilling } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      save,
+      loading,
+      billing
+    } = useBilling();
+
+    const billingForm = ref(/* object for billing details */);
+
+    return {
+      billingForm,
+      save
+    };
+  }
+}
+</script>
+```
+Billing details stored on the server with `save` method will be possible to load after a refresh with `load` method.
+
+## Making an order
+After providing every crucial information by the user, we are ready to *make an order*. To do that, we have to call a `make` method from the `useMakeOrder` composable.
+```js
+import { useMakeOrder } from '{INTEGRATION}';
+
+export default {
+  setup () {
+    const { make } = useMakeOrder();
+
+    return {
+      make
+    }
+  }
+}
+```
+
+It creates an order but we need to perform additional actions:
+- redirect to thank you page
+- clear a cart
```

Comment:
```suggestion
When the order is created, we can redirect the user to the page thanking them for making an order and refresh the cart.
```

`It creates an order but we need to perform additional actions:` sounds like there are some other steps required to finalize the order.